### PR TITLE
#+-13-20% speed up making steps as an alias, and using it in a range

### PR DESCRIPTION
**Pull request by rd4com**
Hello, i am new to github do you have the same results ?

### CODE REVIEW

**Comment by tairov** [comment]
Hi @rd4com .
Thank you for sharing this interesting idea. I’m not certain if it’s related to alias steps, but sometimes I observe a difference in executions within a 10% margin. Maybe it's somehow related to the state of target VM I'm using for tests

File: llama2.mojo [hunk 1]
```diff
@@ -980,10 +975,9 @@ fn main() raises:
...
         # Advance forward
         token = next_token
-        pos += 1
 
         if start == 0:
             start = time_in_ms()
 
     let end = time_in_ms()
-    print("\nachieved tok/s: ", (pos - 1) / (end - start) * 1000)
+    print("\nachieved tok/s: ", (steps - 1) / (end - start) * 1000)
```
Comment (mikowals) [inline llama2.mojo:983]: Why count steps instead of tokens?  Any benchmarking probably needs this line reverted.

Comment (rd4com) [reply llama2.mojo:983]: ah of course... sorry for the waste of time, i believed too much in the power of alias and thought there was a speed up lol that explains it

**Comment by rd4com** [comment]
:vulcan_salute:

**Pull request closed without merging**